client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -123,6 +125,17 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 			}
 		}
 
+		if ( isDesignFirstFlow ) {
+			allDesigns.designs = allDesigns.designs.filter( ( design ) => design.is_premium === false );
+
+			allDesigns.designs = allDesigns.designs.map( ( design ) => {
```

Comment:
Here we're hiding the variation badges, leaving only the default:

![image](https://github.com/Automattic/wp-calypso/assets/1044309/1ca13f95-077a-496c-afeb-13beec9c6d29)